modules/Emails/include/DetailView/EmailsNonImportedDetailView.php

```diff
@@ -73,9 +73,10 @@ public function setup(
     public function populateBean($request = array())
     {
         if (!empty($request['uid']) && !empty($request['inbound_email_record'])&& !empty($request['msgno'])) {
+            /** @var InboundEmail $inboundEmail **/
             $inboundEmail = BeanFactory::getBean('InboundEmail', $request['inbound_email_record']);
```

Comment:
This could be return false.

Reply:
It will only return false it we remove the InboundEmail module from SuiteCRM. The InboundEmail module will still be in SuiteCRM for quite some time.  I will put a check if the bean or it record is missing, and log a warning to the developer.

Reply:
Yeah, and when it might be removed, you will be surprised. You can't be sure it could be renamed either. At least use InboundEmail::class, please.

Reply:
```
if (!$inboundEmail) {
    LoggerManager::getLogger()->warn('Hoops! Something went wrong, when non imported email detail view trying to populate bean and requested inbound email record id was: ' . $request['inbound_email_record']);
}
````